SECURITY.md

```diff
@@ -0,0 +1,5 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+Please report security issues to <email>
```

Comment:
@hacdias, would you like your email address to be included as well?

```suggestion
Please report security issues to:
msaa1990 [at] gmail [dot com]
```

Reply:
Yes, sure. Mention it as a CC.